test/test_specs.py

```diff
@@ -678,6 +685,23 @@ def test_change_batch_size(self, shape, is_complete, device, dtype):
         assert ts["nested"].shape == (3,)
 
 
+class TestChoiceSpec:
+    @pytest.mark.parametrize("input_type", ["spec", "nontensor", "nontensorstack"])
+    def test_choice(self, input_type):
+        if input_type == "spec":
+            stack = torch.stack([Bounded(0, 2.5, ()), Bounded(10, 12, ())])
```

Comment:
do we need to stack or can't we just use a plain list, like
`Choice(spec1, spec2)`?
`stack` implies that there's a dim along which we stack, but the values generated / tested against don't have that concept of stack dim so perhaps stack is superfluous?